corehq/ex-submodules/fluff/pillow.py

```diff
@@ -153,11 +152,30 @@ def get_fluff_pillow(indicator_class, delete_filtered=False):
     processor = FluffPillowProcessor(indicator_class, delete_filtered=delete_filtered)
 
     return FluffPillow(
-        indicator_class=indicator_class,
+        indicator_name=indicator_class.__name__,
+        kafka_topic=indicator_class().kafka_topic,
         processor=processor,
     )
 
 
+def get_multi_fluff_pillow(indicator_classes, name, kafka_topic, delete_filtered=False):
+    processors = [
+        FluffPillowProcessor(cls, delete_filtered=delete_filtered)
+        for cls in indicator_classes
+    ]
+    domains = list(set(d for cls in indicator_classes for d in cls.domains))
+    doc_types = list(set(cls.document_class._doc_type for cls in indicator_classes))
+    assert len(doc_types) == 1
+
+    return FluffPillow(
+        indicator_name=name,
+        kafka_topic=kafka_topic,
+        processor=processors,
```

Comment:
nit: may want to rename to `processors` (though might be more work than it's worth)